src/MIDI.hpp

```diff
@@ -1343,42 +1341,22 @@ void MidiInterface<Transport, Settings, Platform>::launchCallback()
  @{
  */
 
-/*! \brief Set the filter for thru mirroring
- \param inThruFilterMode a filter mode
-
- @see Thru::Mode
- */
-template<class Transport, class Settings, class Platform>
-inline void MidiInterface<Transport, Settings, Platform>::setThruFilterMode(Thru::Mode inThruFilterMode)
-{
-    mThruFilterMode = inThruFilterMode;
-    mThruActivated  = mThruFilterMode != Thru::Off;
-}
-
-template<class Transport, class Settings, class Platform>
-inline Thru::Mode MidiInterface<Transport, Settings, Platform>::getFilterMode() const
-{
-    return mThruFilterMode;
-}
-
 template<class Transport, class Settings, class Platform>
 inline bool MidiInterface<Transport, Settings, Platform>::getThruState() const
 {
-    return mThruActivated;
+    return mThruFilterCallback != thruOff;
 }
```

Comment:
This makes little sense with an external function, as it could point to a `{ return false }` external callback and would still show as active. I think we could get rid of it completely.

Reply:
I think I was just trying to retain as much of the previous behavior as before.  But you're right, we can't really answer whether or not thru is activated anymore.

Reply:
Most of this API was YAGNI, I was learning C++ and eager to provide as much (even if useless) functionality as possible 😅